test/helpers/cilium.go

```diff
@@ -27,6 +28,7 @@ import (
 )
 
 const (
+	//MaxRetries number of times that a loop should try until fails
```

Comment:
Please use complete sentences with code comments. E.g., `MaxRetries is the number of times that a loop should iterate until a specified condition is not met`